client/src/routes/+layout.svelte

```diff
@@ -155,6 +155,21 @@
       <li>Scenario 1 will result in the cats winning. The app is now fish-free!</li>
       <li>Scenario 2 will result in the catfish winning and cats dropping lower in the food chain…</li>
     </ul>
+
+    <h3 class="mdc-typography--headline5">Credits</h3>
+    <a href="https://github.com/nstringham" target="_blank">Nate Stringham - Team Lead</a> <br />
+    <a href="https://github.com/njumaah" target="_blank">Nathan Jumaah - Deputy Team Lead</a> <br />
```

Comment:
Personally I would probably just say "Team Member" for Nathan but I'm fine with whatever

Reply:
This was my position, so I'm going to keep it that way.